plugins/inputs/marklogic/README.md

```diff
@@ -0,0 +1,57 @@
+# MarkLogic Plugin
+
+The MarkLogic Telegraf plugin gathers status metrics from one or more host in a MarkLogic Cluster.
+
+### Configuration:
+
+```toml
+[[inputs.marklogic]]
+  ## Base URL of MarkLogic host for Management API endpoint.
+  url = "http://localhost:8002"
+
+  ## List of specific hostnames in a cluster to retrieve information. At least (1) required.
+  # hosts = ["hostname1", "hostname2"]
+
+  ## Using HTTP Digest Authentication. This requires 'manage-user' role privileges
+  # username = "telegraf"
+  # password = "p@ssw0rd"
+```
+
+### Measurement & Fields:
```

Comment:
Remove this heading and `### Tags:` and add:
```md
### Metrics

- marklogic
  - tags:
    - source (the hostname of the server address, ex. `ml1.local`)
    - id (the host node unique id ex. `2592913110757471141`)
  - fields:
    - online
    - total_load
    - total_rate
    - ncpus
    - ncores
    - total_cpu_stat_user
    - total_cpu_stat_system
    - total_cpu_stat_idle
    - total_cpu_stat_iowait
    - memory_process_size
    - memory_process_rss
    - memory_system_total
    - memory_system_free
    - memory_size
    - host_size
    - data_dir_space
    - query_read_bytes
    - query_read_load
    - http_server_receive_bytes
    - http_server_send_bytes
```

Reply:
@glinton - Thanks for this template